run/pubsub/README.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Run Pub/Sub Tutorial Sample
+
+This sample shows how to create a service that processes Pub/Sub messages.
+
+Use it with the [Cloud Pub/Sub with Cloud Run tutorial](http://cloud.google.com/run/docs/tutorials/pubsub).
+
+## Build
+
+```
+docker build --tag pubsub-tutorial:python .
+```
+
+## Run
+
+```
+docker run --rm -p 9090:8080 pubsub-tutorial:python
+```
+
+## Test
+
+```
+pytest
```

Comment:
Is this preferred over nox?

Reply:
I do not know much about our Python practices. @ace-n do you have insight on this?